Josephus_Problem/Josephus_Problem.java

```diff
@@ -0,0 +1,33 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class Josephus_Problem
+{
+    public static int Josephus(int n, int m) {
+        // base case when only 1 person left return 0 assuming people numbered from 0 to n -1 
+	if (n == 1) {
```

Comment:
In above few lines the indentation is perfect but from this line onwards there is tab indentation (where 1 tab equivalent to ~ 8 spaces).
You have to convert the whole program to incorporate space indentation instead of tabs.